Dashboard/templates/newScan.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+    <meta charset="utf-8">
+
+    <!-- custom css -->
+    <link rel="stylesheet" type="text/css" href="../static/styles/styles.css">
+
+    <!-- Bootstrap css -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <!-- Viewport -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>Scan8: New Scan</title>
+</head>
+
+<body>
+    <div class="container">
+        <h1>Scan8</h1>
+        <button type="button" name="button" class="btn btn-outline-secondary hide">New Scan</button>
+
+        <h4>New Scan</h4>
+        <form action='' method="POST" enctype="multipart/form-data">
+            <div class="custom-file">
+                <input type="file" class="custom-file-input" name="dir" id="customFile" onchange="update()" webkitdirectory mozdirectory>
+                <label class="custom-file-label" for="customFile" id="chooseFileLabel">Choose file</label>
+            </div>
+
+            <div class="container">
+                <div class="row">
+                    <div class="col-sm d-flex justify-content-center">
+                        <a href=""class="btn btn-outline-danger">Cancel</a>
+                    </div>
+                    <div class="col-sm d-flex justify-content-center">
+                        <input class="btn btn-outline-success" type='submit' value='Upload' id="upload" disabled>
+                    </div>
+                </div>
+            </div>
+        </form>
+    </div>
+    
+    <script>
+        function update() {
+            var numFiles = document.getElementById('customFile').files.length;
+            if(numFiles > 0) {
+                document.getElementById('upload').disabled = false;
+            } else {
+                document.getElementById('upload').disabled = true;
+            }
+            var msg;
+            if (numFiles == 0) {
+                msg = "Choose File";
+            } else if (numFiles == 1) {
```

Comment:
@maanas-talwar Why do you need this else if block? Can we remove this?

Reply:
It is just for the sake of "1 file uploaded" and not "1 file(s) uploaded"

Reply:
If the other way suits better I can remove the else if block